troposphere/static/js/components/common/ProjectCreateView.react.js

```diff
@@ -0,0 +1,84 @@
+import React from 'react';
+
+export default React.createClass({
+    displayName: "ProjectCreateModal",
+
+    getInitialState: function () {
+        return {
+            projectName: "",
+            projectDescription: ""
+        };
+    },
+
+    isSubmittable: function(){
+        var hasName = !!this.state.projectName.trim();
+        var hasDescription = !!this.state.projectDescription.trim();
+        return hasName && hasDescription;
+    },
```

Comment:
If this function prevents a submit of an empty projectName - then there is no need to `trim()` in `onProjectCreateViewConfirm`.